lib/web/apps.go

```diff
@@ -300,6 +311,8 @@ type ResolveAppParams struct {
 
 	// AppName is the name of the application
 	AppName string `json:"app_name,omitempty"`
+
+	RequestHostname string
```

Comment:
nit: Godoc

Reply:
this is vestigial actually, so ill just remove!